### DISCUSSION TITLE
After changing the path configuration, do any of the beets commands move the files to their new home?

### DISCUSSION THREAD

**Discussion by Serene-Arc** [beetbox/beets#4539] [Q&A]
As the title says, if you change the path configuration in the config file, do any of the beets commands reorganise the library to conform to the new specification. I tried both the update and move commands on pretend mode and it doesn't seem to accomplish this.

**Comment by wisp3rwind** [comment]
`beet move` should accomplish this. Are you sure that the configuration change is correct? If you post it here and describe the intended effect, we might be able to tell more. You could also try running beets in verbose mode (`beet -vv`), maybe that gives some hints.

**Reply by sampsyo** [reply]
Indeed! As one other tip, we do have an FAQ entry about this situation, in case you hadn't run across it:
https://beets.readthedocs.io/en/stable/faq.html#rename-my-files-according-to-a-new-path-format-configuration

**Reply by Serene-Arc** [reply]
Ah, thanks! It was a special case with the audible plugin, I had the configuration wrong. It's working now! Though the `write` command does seem to be writing the tags every time which is odd. That might be due to the plugin though.
